expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java

```diff
@@ -697,6 +726,17 @@ private static List<Class<?>> listPublicationInstanceTypes() {
         return PublicationInstanceBuilder.listPublicationInstanceTypes();
     }
 
+    private static List<Arguments> licenseProvider() {
+        return List.of(Arguments.of("https://creativecommons.org/licenses/by-nc-nd/2.5", LicenseType.CC_NC_ND),
+                       Arguments.of("https://creativecommons.org/licenses/by-nc-sa/2.0", LicenseType.CC_NC_SA),
+                       Arguments.of("https://creativecommons.org/licenses/by-nc/3.0", LicenseType.CC_NC),
+                       Arguments.of("https://creativecommons.org/licenses/by-nd/4.0", LicenseType.CC_ND),
+                       Arguments.of("https://creativecommons.org/licenses/by-sa/4.0", LicenseType.CC_SA),
+                       Arguments.of("https://creativecommons.org/licenses/by/4.0", LicenseType.CC_BY),
+                       Arguments.of("https://creativecommons.org/unknown/license", LicenseType.OTHER)
```

Comment:
This is all other, not just creativecommons, i.e. '*'

Reply:
`Arguments.of("https://creativecommons.org/unknown/license", LicenseType.OTHER)`
 ->
`  Arguments.of("https://something.else.com", LicenseType.OTHER)`